mage_ai/data_preparation/templates/constants.py

```diff
@@ -206,6 +206,13 @@
         name='Chroma',
         path='data_loaders/chroma.py',
     ),
+    dict(
+        block_type=BlockType.DATA_LOADER,
+        groups=[GROUP_DATABASES],
+        language=BlockLanguage.PYTHON,
+        name='CrateDB',
+        path='data_loaders/cratedb.py',
+    ),
```

Comment:
Ah wow. The framework allows database adapters to be written in other languages than Python?